homeassistant/components/persistent_notification.py

```diff
@@ -74,7 +76,9 @@ def create_service(call):
 
         hass.states.set(entity_id, message, attr)
 
-    hass.services.register(DOMAIN, SERVICE_CREATE, create_service, {},
-                           SCHEMA_SERVICE_CREATE)
+    descriptions = load_yaml_config_file(
+        os.path.join(os.path.dirname(__file__), 'services.yaml'))
```

Comment:
If it moves to its own dir, should I move `persistent_notifications.py` to `persistent_notifications/__init__.py`

In that case maybe rather move under `notify` dir as a platform?